django_postmark_utils/views.py

```diff
@@ -1,20 +1,36 @@
 import json
 import logging
+from functools import wraps
 
 from dateutil import parser
-from django.http import HttpResponse
+from django.conf import settings
+from django.http import HttpResponse, HttpResponseForbidden
 from django.utils.decorators import method_decorator
 from django.utils.translation import ugettext as _
 from django.views import View
 from django.views.decorators.csrf import csrf_exempt
 
 from .models import Bounce, Delivery, Email
 
-
 logger = logging.getLogger(__name__)
 
 
+def url_secret_required(view_func,
+                        correct_secret=settings.POSTMARK_UTILS_SECRET):
+    @wraps(view_func)
+    def _check_secret(request, secret, *args, **kwargs):
+        if secret != correct_secret:
+            logger.error(_("Access forbidden - path: %(request_path)s") % {
```

Comment:
We don't need to log this separately as an error, it isn't really an error, the forbidden response below will be logged and flagged already.